src/components/portfolioPage/Portfolio.jsx

```diff
@@ -55,9 +55,24 @@ class Portfolio extends Component {
     );
   }
 
+  _get_int_update_seq(update_seq){
+    return parseInt(update_seq.slice(0,update_seq.indexOf("-")));
+  }
+
+  async _get_latest_update_seq() {
+    const response = await fetch('http://if05.local');
+    var myjson = await response.json();
+    return myjson.update_seq;
+  }
+
   componentDidMount() {
     let start=new Date().getTime();
     var self=this;
+
+    (async () => {
+      this.update_seq = await this._get_latest_update_seq()
+    })();
```

Comment:
This look weird: you call an async function and do nothing with the result?

Reply:
I think `componentDidMount` cannot be asynchronous.
And I needed to call this function which is asynchronous in the `componentDidMount` function.
So I used this trick.

Reply:
My bad: I didn't see you modified `this.update_seq`.

However I find it risky to set it asynchronously and get it asynchronously with no synchronization between both.

Reply:
Is that better?
```
(async () => {
      let SERVICE = (await conf).services[0];
      const response = await fetch(SERVICE);
      var myjson = await response.json();
      this.update_seq = myjson.update_seq;
    })();
```

Reply:
My comment was more on the global than local level: to be sure that you get the value **after** it was set (or at least assure it would not break if events do not occur in the right order).

However it is hard for me to see the whole picture of how it will look when everything is fixed. So, please, update your commit first (with all the required changes), and I will tell you if it seems OK...